airflow/www_rbac/templates/airflow/version.html

```diff
@@ -24,13 +24,27 @@
 {% block content %}
     {{ super() }}
     <h2>{{ title }}</h2>
-  {% set version_label = 'Version' %}
     {% if airflow_version %}
-        <h4>{{ version_label }} : <a href="https://pypi.python.org/pypi/apache-airflow/{{ airflow_version }}">{{ airflow_version }}</a></h4>
+      {% set airflow_non_local_version = airflow_version.split('.dev')[0].split('+astro')[0] %}
```

Comment:
This is probably complex enough it should be in the view, not in the template -- let's just feed the template the right values it needs.

Reply:
And lets call this variable "airflow_upstream_version"

Reply:
Updated